sdks/java/container/Dockerfile

```diff
@@ -23,10 +23,11 @@ ADD target/slf4j-api.jar /opt/apache/beam/jars/
 ADD target/slf4j-jdk14.jar /opt/apache/beam/jars/
 ADD target/beam-sdks-java-harness.jar /opt/apache/beam/jars/
 
-# Required to run cross-language pipelines with KafkaIO
+# Required to run cross-language pipelines with KafkaIO and SnowflakeIO
 # TODO May be removed once custom environments are supported
 ADD target/beam-sdks-java-io-kafka.jar /opt/apache/beam/jars/
 ADD target/kafka-clients.jar /opt/apache/beam/jars/
+ADD target/beam-sdks-java-io-snowflake.jar /opt/apache/beam/jars/
```

Comment:
Today I learned that there has been added some artifactory which makes it redundant - but I'm not sure whether this is the case here

Reply:
I removed and it works